plugins/woocommerce/composer.json

```diff
@@ -36,7 +36,7 @@
 	],
 	"require": {
 		"php": ">=7.4",
-		"automattic/block-delimiter": "0.2.1",
+		"automattic/block-delimiter": "0.3.4",
```

Comment:
@albarin : I quickly looked through the lock file changes - no red flags. One hint is to check how zip-file size changes with this bump.